addons/website_blog/models/website_blog.py

```diff
@@ -128,9 +128,14 @@ def _compute_website_url(self):
             blog_post.website_url = "/blog/%s/%s" % (slug(blog_post.blog_id), slug(blog_post))
 
     def _default_content(self):
-        return '''
-            <p class="o_default_snippet_text">''' + _("Start writing here...") + '''</p>
+        return f'''
+            <section class="s_text_block" data-snippet="s_text_block">
+                <p class="o_container_small">
+                    {_("Start writing here...")}
+                </div>
```

Comment:
@oussamamessaoudi Did you actually review you work here? :disappointed: A starting p and a closing div...? Anyways, I will make precisions to the specs, this is not supposed to be a 7 lines tasks... :grimacing: